plugin/utils/gapic_utils.py

```diff
@@ -34,7 +34,7 @@
 
 def read_from_gapic_yaml(yaml_file):
     with open(yaml_file) as f:
-        gapic_yaml = yaml.load(f)
+        gapic_yaml = yaml.full_load(f)
```

Comment:
This change is to get around the PyYAML `load()` deprecation: https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation